terraform/modules/aws-dns-records/README.md

```diff
@@ -17,14 +17,21 @@ AWS resources: route53
 module "dns_records" {
   source = "github.com/wireapp/wire-server-deploy.git//terraform/modules/aws-dns-records?ref=CHANGE-ME"
 
-  environment = "staging"
-
   zone_fqdn = "example.com"
+  domain = "staging"
+  sub_domains = [
+    "nginz-https",
```

Comment:
But the only place calling this module is `kubernetes-dns-records` which already defaults to the empty list; so why don't we default to the empty list here? Is it to not break existing usages?

Reply:
Discard this. I thought I was reviewing TF code but this is the README :)